include/nfl/poly_p.hpp

```diff
@@ -141,6 +141,16 @@ class poly_p {
   void ntt_pow_phi() { poly_obj().ntt_pow_phi();}
   void invntt_pow_invphi() { poly_obj().invntt_pow_invphi(); }
 
+  /* set */
+  void set(value_type v, bool reduce_coeffs = true) { poly_obj().set(v, reduce_coeffs); };
+  void set(uniform const& mode) { poly_obj().set(mode); };
+  void set(non_uniform const& mode) { poly_obj().set(mode); };
+  template <class in_class, unsigned _lu_depth> void set(gaussian<in_class, T, _lu_depth> const& mode) { poly_obj().set(mode); };
+  void set(void* mode) { poly_obj().set(mode); };
```

Comment:
It was present in poly.hpp so I added it to poly_p.hpp, but we don't actually use it, so I removed both.